modules/api/src/main/java/org/apache/ignite/catalog/IgniteCatalog.java

```diff
@@ -116,7 +116,7 @@
  *             .ifNotExists()
  *             .partitions(1)
  *             .replicas(3)
- *             .engine(ZoneEngine.AIMEM)
+ *             .storageProfiles("default")
```

Comment:
Please re-read the whole javadoc, seems that we didn't get rid of all mentions of `ENGINE`

Reply:
Fixed